help_to_heat/frontdoor/interface.py

```diff
@@ -164,6 +164,36 @@ def replace_in_session_data(self, session_data):
         return session_data
 
 
+class DuplicateReferralChecker:
```

Comment:
How much effort would it be to have this live in its own file? I think we should try to move away from having these massive 'everything' files.

Reply:
I think that's fine, but interested in what specifically?

Reply:
SupplierConverter references api (in interface.py)
interface.py references SupplierConverter (in Session entity)

could leave SupplierConverter where it is?

Reply:
Yeah I'm happy with that for what it's worth (I think trying to restructure will quickly become a big job as we pull on the strings, for reasons like this - so we're best off just trying to do right with the new things we add for now)